sdk/testing/src/main/java/io/opentelemetry/sdk/testing/exporter/InMemoryMetricReader.java

```diff
@@ -70,6 +88,19 @@ public static InMemoryMetricReader create(
     return new InMemoryMetricReader(aggregationTemporalitySelector, defaultAggregationSelector);
   }
 
+  /**
+   * Returns a new {@link InMemoryMetricReader}.
+   *
+   * @since 1.26.0
+   */
+  public static InMemoryMetricReader create(
+      AggregationTemporalitySelector aggregationTemporalitySelector,
+      DefaultAggregationSelector defaultAggregationSelector,
+      MemoryMode memoryMode) {
```

Comment:
Might be time for a builder instead of adding more overloads.

Reply:
Added a builder and removed the overloaded method. I wasn't sure if I should replace existing factory methods with the builder.